schema/encoding/jsonschema/time.go

```diff
@@ -1,13 +1,12 @@
 package jsonschema
 
-import (
-	"io"
+import "github.com/rs/rest-layer/schema"
 
-	"github.com/rs/rest-layer/schema"
-)
+type timeBuilder schema.Time
 
-func encodeTime(w io.Writer, v *schema.Time) error {
-	ew := errWriter{w: w}
-	ew.writeString(`"type": "string", "format": "date-time"`)
-	return ew.err
+func (v timeBuilder) JSONSchema() (map[string]interface{}, error) {
+	m := make(map[string]interface{})
```

Comment:
That won't quite work. If new fields are added later, they would be added to the global var. E.g. if one timeJSONSchema field is set to be required, all later instances would as well.

I can do what @rs suggests below instead, and do initialization on the allocation line.